pkg/didcomm/common/service/event.go

```diff
@@ -102,8 +102,12 @@ type Event interface {
 //	go service.AutoExecuteActionEvent(actionCh)
 func AutoExecuteActionEvent(ch chan DIDCommAction) error {
 	for msg := range ch {
-		msg.Continue()
+		msg.Continue(&Empty{})
```

Comment:
We should not pass the `Empty` structure here, instead of this I suggest adding an additional parameter to the function signature, we can pass the `Empty` structure on the caller side
e.g `func AutoExecuteActionEvent(ch chan DIDCommAction, opts interface{}) error`

Reply:
This suggestion might need an example for this case.

Reply:
e.g Introduce client keeps agent’s invitation (it is a constant) and we will always pass it to the Continue function, on the service side we can decide whether to use it or no

Reply:
In case of using peer dids only we never need this functionality. Should we have different function for other protocols e.g. AutoExecuteActionEventWithOpts.

Reply:
ok, you can ignore this comment, I will come up with something when it will be needed

Reply:
Let's have a TODO to improve this Empty argument concept.